testnets/junotestnet/assetlist.json

```diff
@@ -23,6 +23,32 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/junotestnet/images/juno.svg"
       },
       "coingecko_id": "juno-network"
+    },
+    {
+      "description": "The native token of NEXX GEN AI",
+      "denom_units": [
+        {
+          "denom": "factory/juno12klaltyqvg2j6v034jwdxrk5n4242ttse4sdpt/NEXX",
+          "exponent": 0,
+          "aliases": [
+            "NEXX"
+          ]
+        },
+        {
+          "denom": "NEXX",
+          "exponent": 6,
+          "aliases": [
+            "factory/juno12klaltyqvg2j6v034jwdxrk5n4242ttse4sdpt/NEXX"
+          ]
+        }
+      ],
+      "base": "factory/juno12klaltyqvg2j6v034jwdxrk5n4242ttse4sdpt/NEXX",
+      "name": "factory/juno12klaltyqvg2j6v034jwdxrk5n4242ttse4sdpt/NEXX",
+      "display": "factory/juno12klaltyqvg2j6v034jwdxrk5n4242ttse4sdpt/NEXX",
```

Comment:
these properties aren't filled in correctly, but the mainnet addition looks correct you could copy some lines from there